tes/models.py

```diff
@@ -266,6 +266,9 @@ class Task(Base):
     description = attrib(
         default=None, converter=strconv, validator=optional(instance_of(str))
     )
+    callback_url = attrib(
```

Comment:
Maybe add a comment that this feature is for something that will be added into the TES specification in the future. Probably in version 1.2

Reply:
Sure